synapse/tools/layer/dump.py

```diff
@@ -0,0 +1,170 @@
+import os
+import sys
+import asyncio
+import argparse
+import tempfile
+
+import synapse.exc as s_exc
+import synapse.common as s_common
+import synapse.telepath as s_telepath
+
+import synapse.lib.coro as s_coro
+import synapse.lib.output as s_output
+import synapse.lib.msgpack as s_msgpack
+import synapse.lib.version as s_version
+
+descr = '''
+Export node edits from a Synapse layer.
+'''
+
+async def exportLayer(opts, outp):
+
+    async with await s_telepath.openurl(opts.url) as cell:
+
+        info = await cell.getCellInfo()
+
+        if (celltype := info['cell']['type']) != 'cortex':
```

Comment:
For future proofing we probably want to make sure we're talking to something that is implementing `CoreApi` via the `_getClasses()` method on the proxy.

Reply:
I'm with blackout on this one. Checking the actual class hierarchy creates overly fragile code which will break if we change the layout or names of classes. I've noticed it a few other places and actually want to move away from it.